### ISSUE TITLE
Travis macOS build errors out while installing gtk+ dependency python

### ISSUE THREAD

**Issue by btj** [verifast/verifast#191]
Travis job [#440.2](https://travis-ci.org/github/verifast/verifast/jobs/649118808) failed:
```
==> Installing gtk+ dependency: python
==> Downloading https://homebrew.bintray.com/bottles/python-3.7.6_1.high_sierra.
==> Downloading from https://akamai.bintray.com/35/3504f29ae0366d08fc025cfe2885c
######################################################################## 100.0%
==> Pouring python-3.7.6_1.high_sierra.bottle.tar.gz
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /usr/local
Could not symlink Frameworks/Python.framework/Headers
Target /usr/local/Frameworks/Python.framework/Headers
is a symlink belonging to python@2. You can unlink it:
  brew unlink python@2

To force the link and overwrite all conflicting files:
  brew link --overwrite python

To list all files that would be deleted:
  brew link --overwrite --dry-run python

Possible conflicting files are:
/usr/local/Frameworks/Python.framework/Headers -> /usr/local/Cellar/python@2/2.7.17/Frameworks/Python.framework/Headers
/usr/local/Frameworks/Python.framework/Python -> /usr/local/Cellar/python@2/2.7.17/Frameworks/Python.framework/Python
/usr/local/Frameworks/Python.framework/Resources -> /usr/local/Cellar/python@2/2.7.17/Frameworks/Python.framework/Resources
/usr/local/Frameworks/Python.framework/Versions/Current -> /usr/local/Cellar/python@2/2.7.17/Frameworks/Python.framework/Versions/Current
```

**Comment by btj** [comment]
See also #127

**Comment by btj** [comment]
Fixed by cb9610e
